app/controllers/instruments_controller.rb

```diff
@@ -9,7 +9,14 @@ class InstrumentsController < ProductsCommonController
   skip_before_action :authenticate_user!, only: [:public_schedule]
   skip_authorize_resource only: [:public_schedule]
 
-  skip_before_action :init_product, only: [:instrument_statuses]
+  skip_before_action :init_product, only: [:instrument_statuses, :list]
+
+  # GET /facilities/:facility_id/instruments/list
+  def list
+    @instruments = Instrument.active.in_active_facility.order(:name).includes(:facility)
```

Comment:
I think you could do `joins(:facility).merge(Facility.active)` instead. I'm not sure how valuable the scope is.